admin/tests/test_unit/test_routes/urls.py

```diff
@@ -0,0 +1,4 @@
+ADMIN_LOGIN = '/admin_login'
+HOME = '/'
+BOOK_STORAGE = '/book-storage'
+UPDATE_USER = '/update_user'
```

Comment:
Maybe we should sort the paths alphabetically in this file?
Just my suggestion, is not my requirement.